util/travis/linux-cibuild.cmake

```diff
@@ -36,7 +36,7 @@ CONFIGURE_FILE (${CTEST_SOURCE_DIRECTORY}/util/cmake/CTestConfig.cmake
 
 # run the classical ctest suite without update
 # travis-ci handles this for us
-ctest_start     (Continuous)
+ctest_start     (Continuous TRACK "Continuous-$ENV{TRAVIS_BRANCH}")
```

Comment:
related?

Reply:
Not directly but didn’t want to make an extra PR for this